src/util/envVariablesValidations.ts

```diff
@@ -36,6 +36,15 @@ const checkExpectedVariables = () => {
   return true;
 };
 
+const checkUnexpectedVariables = () => {
+  for (const v of Object.keys(process.env)) {
+    if (!expectedVariables.includes(v)) {
+      throw new Error(`Unexpected ${v} variable found.`);
+    }
```

Comment:
I'm not sure if this is the right way to solve the issue, wouldn't `process.env` contain some random stuff? Or are the system environment variables filtered (i.e. only `REACT_APP` variables are getting in)? If the latter then it's good :)

Reply:
This test ensures that all variables set in .env file provided in the repo will be test agains (so the `REACT_APP` variables). I can always update to check for the specific ones onl y (Start with REACT_APP) but i think as soon as the process.env is loaded inside the scope of this repo - no random stuff should exist